lib/3scale/core/application.rb

```diff
@@ -25,7 +25,7 @@ def self.app_uri(service_id, id)
       private_class_method :app_uri
 
       def self.key_uri(service_id, key)
-        escaped_key = CGI::escape(key)
+        escaped_key = CGI::escape(key.to_s)
```

Comment:
I wonder whether we should use `ERB::Util.url_encode` or `CGI::escape`.

```
[1] pry(main)> str = "asdasd asdas"
[3] pry(main)> require 'cgi'
[6] pry(main)> require 'erb'

[4] pry(main)> CGI.escape str
=> "asdasd+asdas"

[7] pry(main)> ERB::Util.url_encode str
=> "asdasd%20asdas"
```

I'm not sure what is the processing on the other side. So this is just a question. Not an objection.